src/Cthulhu.jl

```diff
@@ -174,15 +174,18 @@ function lookup(interp::CthulhuInterpreter, mi::MethodInstance, optimize::Bool)
         rt = interp.unopt[mi].rt
         infos = interp.unopt[mi].stmt_infos
         slottypes = codeinf.slottypes
+        if isnothing(slottypes)
+            slottypes = Any[ Any for i = 1:nslots ]
```

Comment:
`nslots` does not seem to be defined anywhere

Reply:
ops I just forgot to push the local fix. Thanks.